docs/reference/settings.rst

```diff
@@ -246,6 +246,75 @@ subclass.
 See :ref:`retry`.
 
 
+.. setting:: ZYTE_API_SESSION_CHECKER
+
+ZYTE_API_SESSION_CHECKER
+========================
+
+Default: ``None``
+
+A :ref:`Scrapy component <topics-components>` (or its import path as a string)
+that defines a ``check`` function.
+
+When defined, :ref:`sessions managed by scrapy-zyte-api <plugin-sessions>` are
+enabled, and the ``check`` function is called on every response. If ``check``
+returns ``True``, the response session is considered valid; if ``check``
+returns ``False``, the response session is considered invalid, and will be
+expired.
+
+For example:
+
+.. code-block:: python
+
+    class MySessionChecker:
+
+        def check(self, request, response):
+            return bool(response.css(".is_valid"))
+
+
+.. setting:: ZYTE_API_SESSION_COUNT
+
+ZYTE_API_SESSION_COUNT
+======================
+
+Default: ``8``
```

Comment:
Is there a limit on how much RPM can a single session handle? If so, it'd be nice to have a hint on how to determine needed session count.

Reply:
> Is there a limit on how much RPM can a single session handle?

I don’t think Zyte API imposes one at the moment, based on my own experiments.

> it'd be nice to have a hint on how to determine needed session count.

Other than trial and error, I cannot think of a reliable way.